tests/Oryx.Integration.Tests/DotNetCore/DotNetCoreRuntimeVersion60Tests.cs

```diff
@@ -71,6 +71,122 @@ public async Task CanBuildAndRun_NetCore60MvcApp()
                 });
         }
 
+        [Fact]
+        public async Task CanBuildAndRun_Adds_Oryx_AppInsights_Codeless_Configuration()
+        {
+            // Arrange
+            var dotnetcoreVersion = DotNetCoreRunTimeVersions.NetCoreApp60;
+            var hostDir = Path.Combine(_hostSamplesDir, "DotNetCore", NetCoreApp60MvcApp);
+            var volume = DockerVolume.CreateMirror(hostDir);
+            var appDir = volume.ContainerDir;
+            var appOutputDirVolume = CreateAppOutputDirVolume();
+            var appOutputDir = appOutputDirVolume.ContainerDir;
+            var agentExtensionVersionEnv = ExtVarNames.UserAppInsightsAgentExtensionVersion;
+            var connectionStringEnv = ExtVarNames.UserAppInsightsConnectionStringEnv;
+            var OryxAppInsightsAttachString1 = "export ASPNETCORE_HOSTINGSTARTUPASSEMBLIES=";
+            var OryxAppInsightsAttachString2 = "export DOTNET_STARTUP_HOOKS=";
+            
+            var buildImageScript = new ShellScriptBuilder()
+               .AddDefaultTestEnvironmentVariables()
+               .AddCommand(
+                $"oryx build {appDir} -i /tmp/int --platform {DotNetCoreConstants.PlatformName} " +
+                $"--platform-version {dotnetcoreVersion} -o {appOutputDir}")
+               .ToString();
+            var runtimeImageScript = new ShellScriptBuilder()
+                .AddCommand($"export {agentExtensionVersionEnv}=~3")
+                .AddCommand($"export {connectionStringEnv}=alkajsldkajd")
```

Comment:
Hey Arjun, I mean the commands here we don't need to manually set it up right? This is a positive test, as users they don't need to add these command when they run the app.. so that's my point

Reply:
these 2 env we need to set as they are passed to us by appservice